civis/io/_tables.py

```diff
@@ -255,6 +257,20 @@ def civis_to_csv(filename, sql, database, job_name=None, api_key=None,
     credential_id : str or int, optional
         The ID of the database credential.  If ``None``, the default
         credential will be used.
+    include_header: bool, optional
+        If ``True`` include a key in the returned dictionary containing a list
+        of column names. Default: ``True``.
+    compression: str, optional
+        Type of compression to use, if any. One of ``'none'``, ``'zip'``, or
+        ``'gzip'``. Default ``'none'``.
+    delimiter, str: optional
+        Which delimiter to use, if any. One of ``','``, ``'\t'``, or
+        ``'|'``. Default: ``','``.
+    unquoted: bool, optional
+        Where or not to quote fields. Default: ``False``.
+    prefix: str, optional
+        A user specified filename prefix for the output file to have. Default:
+        ``None``.
```

Comment:
Is this only used internally in the Platform? Might be a bit confusing, since the user specifies a `filename` directly (and this doesn't affect that).